.github/workflows/openbsd.yaml

```diff
@@ -0,0 +1,66 @@
+name: Staged commit - OpenBSD
+
+on:
+  push:
+    branches: [ "bsd-merge-tests" ]
```

Comment:
This is the name of a PR branch. Another branch ought to be used here.

Why are we creating openbsd.yaml instead of reusing slow.yaml or quick.yaml where these and similar decisions have been made and corrected already?

Reply:
Unfortunately you reviewed an obsolete commit; this had been deleted already.